uspace/app/bdsh/cmds/modules/ls/ls.c

```diff
@@ -109,32 +109,33 @@ static bool ls_print(struct dir_elem_t *de)
 			cap_simplify(&cap);
 
 			char *rptr;
-			if (cap_format(&cap, &rptr) == EOK) {
+			errno_t rc = cap_format(&cap, &rptr);
+			if (rc == EOK) {
```

Comment:
Could you also invert this condition so that the following code isn't unnecessarily indented? Thanks!